### ISSUE TITLE
Plots do not show

### ISSUE THREAD

**Issue by guiloma** [poliastro/poliastro#142]
After performing a basic installation,  I have tried first example at http://poliastro.readthedocs.io/en/latest/user_guide.html, having no errors but the plot is not displayed.

After some research it seems that "plt.show()" has to be called in order to display the graph.

Shouldn't that method be called from the plot() function?

**Comment by OrbitalMechanic** [comment]
Attached is a copy of a program that I hope me be of help to you. In the spirit of full disclosure, I'm still a novice when it comes to Python, so the code may seem simple, but it does the same thing as in the example only that I've specified the state vector and its epoch. The program runs under Python 2.7.12 under the Anaconda Distribution 4.2.0. The output plot in .png format is also provided.

Hope this helps.


![two-body_trajectory_2d-view](https://cloud.githubusercontent.com/assets/4779511/22407959/ad8c67b0-e63e-11e6-978f-8c91aede8618.png)
[Simple-Keplerian-Trajectory-Plot.py.txt](https://github.com/poliastro/poliastro/files/737808/Simple-Keplerian-Trajectory-Plot.py.txt)

**Comment by Juanlu001** [comment]
Thanks for the bug report @guiloma and sorry for the long delay. I took the freedom of editing @OrbitalMechanic comment and replacing the attached file with a gist, so the example is easier to read.

Regarding the examples, as it's stated [in the documentation](http://poliastro.readthedocs.io/en/v0.5.0/user_guide.html#from-position-and-velocity):

> If we’re working on interactive mode (for example, using the wonderful IPython notebook) we can immediately plot the current state:

The problem with including a `plt.show()` call inside poliastro plotting functions is that it effectively blocks the whole program until the script finishes and then all the plot information disappears. For example, this prevents the user from modifying the axes and other properties after calling the function. I prefer to keep this bit outside the API.

However, as I have noticed that the library is attracting many Python novices, I just added a warning in the docs so it's clearer that extra code is needed in case we are not working on interactive mode:

http://poliastro.readthedocs.io/en/latest/user_guide.html#from-position-and-velocity

Thanks again to both of you!
